From 3788939047d3450668e74c26c51f9091fcf05522 Mon Sep 17 00:00:00 2001
From: Gabriel Rauter <rauter.gabriel@gmail.com>
Date: Tue, 4 May 2021 11:16:28 +0200
Subject: [PATCH 2/2] Build GUI application on windows

Add win_subsystem: 'windows' keyword argument to the baobab executable
in meson so it builds a GUI instead of a TUI on Windows.
Needs meson >= 0.56
---
 src/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/meson.build b/src/meson.build
index b857f3d..d094418 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -49,5 +49,6 @@ executable('baobab', baobab_sources,
   c_args: baobab_c_args,
   dependencies: baobab_dependencies,
   export_dynamic: true,
+  win_subsystem: 'windows',
   install: true
 )
-- 
2.31.1

